exp/api/v1beta1/azuremachinepool_webhook.go

```diff
@@ -31,7 +31,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
 	"sigs.k8s.io/cluster-api-provider-azure/feature"
-	"sigs.k8s.io/cluster-api-provider-azure/util/azure"
+	azureutil "sigs.k8s.io/cluster-api-provider-azure/util/azure"
```

Comment:
It seems like this import isn't used. I think you might need to find where we called `azure.something` and replace it with `azureutil.something`.